src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -2750,6 +2750,34 @@ difficulty_type BlockchainLMDB::get_block_difficulty(const uint64_t& height) con
   return diff1 - diff2;
 }
 
+void BlockchainLMDB::overwrite_block_cumulative_difficulty(const uint64_t& height, const difficulty_type& new_cumulative_difficulty)
+{
+  LOG_PRINT_L3("BlockchainLMDB::" << __func__);
+  check_open();
+  mdb_txn_cursors *m_cursors = &m_wcursors;
+
+  int result = 0;
+  block_wtxn_start();
```

Comment:
Does the existing TXN_BLOCK_PREFIX macro not work here ?

Reply:
I'm not very familiar with LMDB operations, but it didn't work when it replaced it with `TXN_BLOCK_PREFIX(0)` or `TXN_PREFIX(0)`.